index.csv

```diff
@@ -194,7 +198,7 @@ slug,MIT,0.4.1,0.4.2,0.5.0,0.6.0,0.7.0,0.7.1,0.8.0,0.9.0,0.9.1,0.9.2,0.9.3
 soap,MIT,0.0.2,0.0.3,0.0.7,0.0.8,0.0.9,0.0.10,0.0.11,0.1.0,0.1.1,0.1.2,0.1.3,0.2.0,0.2.1,0.2.2,0.2.3,0.2.4,0.2.6,0.2.7,0.3.0,0.3.1,0.3.2,0.4.0,0.4.2,0.4.4,0.4.5,0.4.6,0.4.7,0.5.0,0.5.1,0.6.0,0.6.1,0.7.0,0.8.0,0.9.0
 socket.io,MIT,0.3.8,0.4.0,0.4.1,0.5.1,0.5.3,0.6.0,0.6.1,0.6.3,0.6.4,0.6.5,0.6.6,0.6.7,0.6.8,0.6.9,0.6.10,0.6.11,0.6.12,0.6.14,0.6.15,0.6.16,0.6.17,0.6.18
 source-map-loader,MIT,0.1.0,0.1.1,0.1.2,0.1.3,0.1.4,0.1.5,0.1.6,0.2.0,0.2.1,0.2.2,0.2.3
-source-map-support,MIT,0.2.9,0.2.10
+source-map-support,MIT,0.1.0,0.1.1,0.1.2,0.1.3,0.1.4,0.1.5,0.1.6,0.1.7,0.1.8,0.1.9,0.2.0,0.2.1,0.2.2,0.2.3,0.2.4,0.2.5,0.2.6,0.2.7,0.2.8,0.2.9,0.2.10,0.3.0,0.3.1,0.3.2
```

Comment:
cc @evanw @julien-f @LinusU

Reply:
These versions where before I joined the project so I'm afraid I can't help here...